doc/modules/preprocessing.rst

```diff
@@ -590,6 +590,21 @@ In the transformed `X`, the first column is the encoding of the feature with
 categories "male"/"female", while the remaining 6 columns is the encoding of
 the 2 features with respectively 3 categories each.
 
+:class:`OneHotEncoder` supports categorical features with missing values by
+considering the missing values as an additonal category::
+
+    >>> X = [['male', 'Safari'],
+    ...      ['female', None],
+    ...      [np.nan, 'Firefox']]
+    >>> enc = preprocessing.OneHotEncoder().fit(X)
+    >>> enc.categories_
+    [array(['female', 'male', nan], dtype=object),
+     array(['Firefox', 'Safari', None], dtype=object)]
+    >>> enc.transform(X).toarray()
+    array([[0., 1., 0., 0., 1., 0.],
+           [1., 0., 0., 0., 0., 1.],
+           [0., 0., 1., 1., 0., 0.]])
```

Comment:
can you add a note here about what happens if None and np.nan are present in the same column?

Reply:
I would also pass explicitly here the handle_unknown parameter. To document it.

Reply:
Updated PR:

1. The user guide now describes what happens when `nan` and `None` are present.
2. The default value of `handle_unknown='error'` is passed explicitly.